2.0/conn/db_engine.py

```
@@ -0,0 +1,6 @@
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+from config.settings import SQL_DATABASE_URL
+
+engine = create_engine(SQL_DATABASE_URL) #echo=True FOR DEBUG ONLY
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)